### ISSUE TITLE
Feedback for "SAP Usage Data Management Service - Service Plans"

### ISSUE THREAD

**Issue by I327556** [SAP-docs/btp-cloud-platform#104]
https://help.sap.com/docs/BTP/65de2977205c403bbc107264b8eccf4b/c94c85ee026846f1843e3bc9abd111f1.html

Hi Colleagues,

There is a customer case where the customer was referring for this documentation. Here you can find the case:
**[REDACTED by ORG owner]** / Service broker error: Service broker uas-broker failed with: 404 Not Found:

So the customer got an error and it seems to be there is a prerequisite that is not mentioned in this document:
After entering the Cockpit, under the account explorer section, select Directories and Subaccount.  Then the desired directory has to be chosen. After click on the tile Users. Here select the button "Enable Entitlement and User Management."

Thank you and best regards,
Szabolcs

**Comment by DavidMitreski5** [comment]
Thank you for your feedback! We’ll look into it and come back to you if we have any questions.

**Comment by SarahErdmann** [comment]
Hi @I327556  Szabolcs,
  
Thank you for pointing this out! 
Unfortunately I can't open/ view the case :
[516561/2022](https://sap.service-now.com/now/workspace/agent/record/sn_customerservice_case/911d44a11b305954f35162406b4bcb78). I assume this is specifically about the "**reporting-directory**" service?  If you want to use the service plan "reporting-directory" to generate reports for a directory, user management has to be enabled for this directory.”?

Kind regards,
Sarah

**Comment by I327556** [comment]
Hi Sarah,

Sorry, it seems to be there was an issue with the link. Anyway yes, the topic is related to the reporting-directory. I think this is the case. But I would recommend clarifying it with a developer before changing the document or I can also contact the one who worked on the case.

Best regards,
Szabolcs

**Comment by DavidMitreski5** [comment]
Hi @SarahErdmann, do we have any updates on this issue?

**Comment by SarahErdmann** [comment]
Hi @I327556 and @DavidMitreski5,

I'm sorry for my late reply. 
I added "**User management must be enabled for the directory, see [Manage Users in Directories [Feature Set B]](https://help.sap.com/docs/BTP/65de2977205c403bbc107264b8eccf4b/ff4d4a4caff94b0486b6427eaa8a0b91.html?state=DRAFT&version=Dev).**" to the description of the reporting-directory service plan on the Help Portal page [SAP Usage Data Management Service - Service Plans](https://help.sap.com/docs/BTP/65de2977205c403bbc107264b8eccf4b/c94c85ee026846f1843e3bc9abd111f1.html?state=DRAFT&version=Dev). 
(I think this short text is better than giving a click-level description of how to enable entitlement and user management for a directory. The UI/navigation in the cockpit can be subject to change in the future.)
I got the OK from development for this change and the change will be published with the next update of the Help Portal documentation in November.
